assets/scripts/elfish.js

```diff
@@ -293,8 +293,63 @@ function exportCSV () {
     return csv;
 }
 
+function computeValue(s,g,e,vals) {
+    var arr = [];
+    var postfix = "-" + s + "-" + g + "-" + e;
+    var t = 0;
+    for (var i = 0; i < vals.length; i++) {
+        var val = vals[i];
+        if (val === "") {
+            console.log("Clearing innerHTML");
+            document.getElementById("est" + postfix).innerHTML = "---";
```

Comment:
Maybe clearEst(s, g, e); clearKe(s, g, e); clearTe(s, g, e)? Or just clearEffort(s,g,e);? A bit smelly doing DOM stuff in computeValue().